WearComplicationProvidersTestSuite/Wearable/src/main/java/com/example/android/wearable/wear/wearcomplicationproviderstestsuite/LongTextProviderService.kt

```diff
@@ -19,11 +19,21 @@ import android.content.ComponentName
 import android.graphics.drawable.Icon
 import android.support.wearable.complications.ComplicationData
 import android.support.wearable.complications.ComplicationManager
+import android.support.wearable.complications.ComplicationProviderService
 import android.support.wearable.complications.ComplicationText
+import androidx.datastore.core.DataStore
 
 /**
  * A complication provider that supports only [ComplicationData.TYPE_LONG_TEXT] and cycles
  * through the possible configurations on tap.
+ *
+ * Note: This subclasses [ComplicationProviderService] to support coroutines, so data operations
```

Comment:
Maybe reword? Technically, this class doesn't subclass ComplicationProviderService. Maybe...

Note: [SuspendingComplicationProviderService] adds coroutine support to [ComplicationProviderService] so data operations (specifically, calls to [DataStore]) can be supported directly in the [onComplicationUpdate].
 See [SuspendingComplicationProviderService] for the implementation details... 

then keep the rest?

Reply:
Whoops, that was a typo! Reworded in the latest commit.